Sources/Components/Callout/CalloutView.swift

```diff
@@ -5,6 +5,23 @@
 import UIKit
 
 public final class CalloutView: UIView {
+
+    public enum Direction: CaseIterable {
+        case up
+        case down
+    }
+
+    private let direction: Direction
+
+    private var arrowRotationTransformation: CGAffineTransform? {
+        switch direction {
+        case .up:
+            return nil
```

Comment:
How about returning `.identity` here? So we don't have to check for `nil`-values